saos-persistence/src/main/java/pl/edu/icm/saos/persistence/repository/JudgmentRepositoryCustomImpl.java

```diff
@@ -56,6 +60,29 @@
         return retJudgment;
         
     }
+    
+    @Override
+    @Transactional
+    public Map<Long, Long> countReferencingJudgmentsForNotIndexed() {
```

Comment:
I wonder it this method shouldn't return 0 (zero) for not indexed judgments that have no referencing judgments. This way typical count works. Now this method just returns only non-zero counts.